optuna/integration/lightgbm_tuner/optimize.py

```diff
@@ -319,8 +346,11 @@ def __init__(
         sample_size=None,  # type: Optional[int]
         best_params=None,  # type: Optional[Dict[str, Any]]
```

Comment:
Not a strong opinion but maybe we can deprecate this as well. You can seemingly get the params via `lightgbm.basic.Booster.params` (key-value dictionary).

Reply:
Thank you for your suggestion. I  added the deprecation warning in https://github.com/optuna/optuna/pull/1032/commits/894a666c774e2b27f1811efad1e853c6add77cb3.